inc/Engine/WPRocketUninstall.php

```diff
@@ -40,6 +40,7 @@ class WPRocketUninstall {
 		'wp_rocket_no_licence',
 		'wp_rocket_last_option_hash',
 		'wp_rocket_debug',
+		'rocketcdn_old_url',
```

Comment:
the way you use the option, it's not a new one, but one that is part of the wp_rocket_settings array.

I think this should not be part of it, so you would need to use options_api only for managing the option, not Options_Data

Reply:
That's a leftover from the old approach, removed